daffodil-test/src/test/scala-debug/org/apache/daffodil/section05/simple_types/TestSimpleTypesDebug.scala

```diff
@@ -98,11 +98,4 @@ class TestSimpleTypesDebug {
   @Test def test_bitOrderChangeInvalid3() { runner1.runOneTest("bitOrderChangeInvalid3") }
 
   @Test def test_bitOrderChangeInvalid() { runner3.runOneTest("bitOrderChangeInvalid") }
-
-  //////////////////////// DAFFODIL-1945 /////////////////////////////
-  @Test def test_dateCalendarDaysInFirstWeek3() { runner.runOneTest("dateCalendarDaysInFirstWeek3") }
-  @Test def test_dateCalendarDaysInFirstWeek5() { runner.runOneTest("dateCalendarDaysInFirstWeek5") }
```

Comment:
These two tests are removed from scala-debug but not added to scala.

Reply:
They were already added from a previous commit, but the debug tests were never removed.

Reply:
Are you sure? I only see them commented out in TestSimpleTypes.scala, lines 113 and 115.

Reply:
You are right, I could have sworn I enabled them, but I guess I didn't.  I was thinking of the "FirstDayOfWeek" tests that were already enabled, not the "DaysInFirstWeek".

Reply:
Responding to steve's original comment..... Ok, so we need to contact the original user (per email thread in the DAFFODIL-1945 ticket) and find out what operating system, and what locale and time zone, etc. It's possible race conditions caused this transient failure, in which case keeping the tests commented out isn't helpful at finding the problem, but we need to try to reproduce.